__tests__/__snapshots__/ordCdsrc.test.js.snap

```diff
@@ -5,11 +5,13 @@ exports[`Tests for default ORD document when .cdsrc.json is present Successfully
   "$schema": "https://sap.github.io/open-resource-discovery/spec-v1/interfaces/Document.schema.json",
   "apiResources": [
     {
-      "apiProtocol": "odata-v4",
-      "description": "AdminService",
+      "apiProtocol": "odata-v5",
+      "description": "new AdminService",
       "entityTypeMappings": [
         {
-          "entityTypeTargets": [],
+          "entityTypeTargets": [
+            "new.entityType.target",
```

Comment:
This is also not compliant to ORD spec, it's actually an object with `ordId` of the target entity type


```suggestion
            {"ordId": ""sap.odm:entityType:BusinessPartner:v1"},
```

Reply:
In the [document](https://pages.github.tools.sap/CentralEngineering/open-resource-discovery-specification/spec-v1/interfaces/document#entity-type-mapping_entitytypetargets), it shows the type is Array<[Entity Type Target (ORD ID)](https://pages.github.tools.sap/CentralEngineering/open-resource-discovery-specification/spec-v1/interfaces/document#entity-type-target-ord-id) | [Entity Type Target (Correlation ID)](https://pages.github.tools.sap/CentralEngineering/open-resource-discovery-specification/spec-v1/interfaces/document#entity-type-target-correlation-id)>. Is there update?